docs/medical-api/api-reference/document/get-document.mdx

```diff
@@ -1,28 +1,20 @@
 ---
 title: "Get Document"
-description: "Downloads the Document contents of the specified URL."
-api: "GET /medical/v1/document/download"
+description: "Fetches the document and sends a presigned URL"
+api: "GET /medical/v1/document/downloadUrl"
 ---
 
-## Query Params
-
-<ParamField query="patientId" type="string" required>
-  The ID of the Patient for which to list available Documents.
-</ParamField>
+This endpoint returns a URL which you can use to download the specified document using the file name provided from the [List Documents endpoint](/medical-api/api-reference/document/list-documents)
```

Comment:
hyper nit: missing `.`

Reply:
Should we also add a warning that the presigned URL will only be valid for 1 minute (or whatever the duration was)?

Reply:
Good point. I suggested we don't return a URL, but return a redirect to the S3 presigned URL... in which case the doc can be updated to state there's a redirect on the response. OTOH, if we decide to keep returning the URL, I think we might need to increase the expiring duration from 20s to something close to 1min - who knows, they might want to have a queue between getting the URL and actual downloading the file.

But I would go with the redirect instead of returning a URL. 😄

Reply:
Let's leave as is and try to build the redirect later on https://github.com/metriport/metriport-internal/issues/609

Reply:
So, back to Dima's comments:
- hyper nit: missing .
- add a warning that the presigned URL will only be valid for 1 minute (or whatever the duration was)